wlxwzx-JWTT/cloudfunctions/TTSService/config.py

```
@@ -0,0 +1,101 @@
+    ],
+    
+    # 需要清理的字符
+    'CLEANUP_CHARS': [
+        '\r', '\t', '\xa0',  # 特殊空白字符
+        '​', '‌', '‍',         # 零宽字符
+        '﻿',                  # BOM字符
+    ],
+    
+    # 需要规范化的标点符号
+    'PUNCTUATION_NORMALIZE': {
+        '，': '，',  # 全角逗号
```

Ruff (0.13.3)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)